packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -313,12 +313,25 @@ export default class RunPanelStore {
     @action.bound
     showErrorMessage(data) {
         const { journal } = this.root_store;
+
         journal.onError(data);
         if (journal.journal_filters.some(filter => filter === message_types.ERROR)) {
             this.setActiveTabIndex(2);
+        } else {
+            this.root_store.ui.addNotificationMessage(journalError(this.switchToJournal));
         }
     }
 
+    @action.bound
+    switchToJournal = () => {
+        const { journal } = this.root_store;
+        journal.journal_filters.push(message_types.ERROR);
+        this.setActiveTabIndex(2);
+        this.root_store.ui.toggleNotificationsModal();
+        this.root_store.ui.removeNotificationMessage({ key: 'bot_error' });
+        this.root_store.ui.removeNotificationByKey({ key: 'bot_error' });
```

Comment:
Does this do the same thing?

Reply:
The `removeNotificationMessage` function removes the notification from the `notification_messages` array which needs to be removed for future errors to be added to the array but this doesn't remove it from the notification center. The `removeNotificationByKey` function removes the notification from the `notifications` array which removes it from the notification center. Hence, both are required. (I hope this was clear. 😅 )